src/cluster.h

```diff
@@ -39,7 +42,8 @@ typedef struct clusterLink {
     char *rcvbuf;               /* Packet reception buffer */
     size_t rcvbuf_len;          /* Used size of rcvbuf */
     size_t rcvbuf_alloc;        /* Allocated size of rcvbuf */
-    struct clusterNode *node;   /* Node related to this link if any, or NULL */
+    struct clusterNode *node;   /* Node related to this link. Initialized to NULL when unknown */
+    int inbound;                /* 1 if this link is accepted from the related node */
```

Comment:
Phrasing for the documentation is a little weird, it sounds like this is set to 1 when the connection has been accepted, not that it's inbound. 

Also, is the inbound bit strictly needed for correctness?